packages/web-components/src/components/card/__stories__/card.stories.ts

```diff
@@ -16,6 +16,7 @@ import ifNonNull from 'carbon-web-components/es/globals/directives/if-non-null';
 import readme from './README.stories.mdx';
 import textNullable from '../../../../.storybook/knob-text-nullable';
 import '../card';
+import { PICTOGRAM_POSITION } from '../defs';
```

Comment:
To align to `ICON_PLACEMENT` for `<dds-card-footer>`:

```suggestion
import { PICTOGRAM_PLACEMENT } from '../defs';
```